Regression/WarpX-tests.ini

```
@@ -2508,6 +2508,23 @@ doVis = 0
 compareParticles = 0
 analysisRoutine = Examples/Tests/ElectrostaticSphere/analysis_electrostatic_sphere.py
 
+[FieldProbe]
+buildDir = .
+inputFile = Examples/Tests/FieldProbe/inputs_2d
+runtime_params =
+dim = 2
+addToCompileString = USE_EB=TRUE USE_OPENPMD=TRUE
+cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_EB=ON -DWarpX_OPENPMD=ON
```

openPMD is not yet used in this test:
```suggestion
addToCompileString = USE_EB=TRUE
cmakeSetupOpts = -DWarpX_DIMS=2 -DWarpX_EB=ON
```